ccd-definition/FixedLists.json

```diff
@@ -513,5 +513,89 @@
     "ID": "PartyType",
     "ListElementCode": "ORGANISATION",
     "ListElement": "Organisation"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
```

Comment:
I think the word 'list' is redundant in this file (fixedLists). `hearingType` is enough in my opinion.

Reply:
All of the other lists finish in the word list so I want to keep it consistent so I think it should remain HearingTypeList ? @Jamesnelson117

Reply:
James, please resolve conversation if you are happy with Becky comment.

Reply:
HearingType please

Reply:
@rebeccabrennan HearingType please